import/index_java/src/org/solrmarc/index/VuFindIndexer.java

```diff
@@ -1624,19 +1631,136 @@ protected Double coordinateToDecimal(String coordinateStr) {
      * @return boolean
      */
     protected boolean validateCoordinates(Double west, Double east, Double north, Double south) {
-        if (west == null || east == null || north == null || south == null) {
-            return false;
+        String validLines = "";
```

Comment:
Is there a specific reason that you chose String types here rather than booleans?